3581/3/app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,14 @@
+class ReviewController < ApplicationController
+  post '/reviews/' do
```

Comment:
if you pass non-exists place id - there is will be exception. add before filter to check the exception